### ISSUE TITLE
GPU installation question.

### ISSUE THREAD

**Issue by salvatoreNRIM** [DeepLabCut/DeepLabCut#524]
Windows 10, deeplabcut 2.1.3, NVIDIA driver 441.41,  CUDA 9. Using Jupyter on firefox.

When running DLC function 'train_network', it looks like it is running (printing out 'training network'). However, in task manager the GPU is not used. I tried to run it on the Reaching example video for over a day, but it does not start the training.

**Comment by MMathisLab** [comment]
You likely do not have your GPU installed correctly then. Please also fill out the templates fully in the future. Please see these tips: https://github.com/AlexEMG/DeepLabCut/blob/master/docs/installation.md#the-most-common-new-user-hurdle-is-installing-and-using-your-gpu

**Comment by salvatoreNRIM** [comment]
Thanks for the answer. I have re-followed the instructions and trying with different versions of CUDA and drivers. I have tried with the recommended CUDA 9 with Nvidia driver 384, but then I ran into the issue 'InternalError: cudaGetDevice() failed. Status: CUDA driver version is insufficient for CUDA runtime version'. 
Given that my GPU is Nvidia Quadro M1000M, I tried with later drivers compatible with it, such as 411.63, but then I again ran into the first issue: DLC is not able to run the network on the GPU. 
So to test the CUDA installation I also ran in python (as suggested in the guide)
`import tensorflow as tf`
`sess = tf.Session(config=tf.ConfigProto(log_device_placement=True))`
And this is what I get 


![Screenshot (7)](https://user-images.githubusercontent.com/46469676/70698796-7ef43200-1d0b-11ea-8a16-0b1692592af7.png)
